doc/content/microreactors/airjacket/index.md

```diff
@@ -0,0 +1,502 @@
+and correlations. In this case, we will find that assuming the flow is not laminar is appropriate. Also note that the main unknowns are still $\bar{U}$ and $T_m$ or $T_o$,
+ with [derived_qty]-[derived_qty_end] being secondary equations that will feed into our main pressure and energy balance equations.
+
+The pressure balance equation has already been discussed ([p_balance]). The energy balance is given by the balance between the convective heat transfer between the hot wall and the fluid bulk, and the heat advected into and out of the
+air jacket:
+\begin{equation}
+q_{hot wall} = q_{advection}
+\end{equation}
+\begin{equation}
+A_{hw} h(T_{hot} - T_{mean}) = \dot{m} C_p (T_o - T_{in})
+\end{equation}
+\begin{equation}
+A_{hw} h(T_{hot} - T_{mean}) = \rho_{mean} \bar{U} A_c C_p (T_o - T_{in}) \label{q_balance}
+\end{equation}
+where $A_{hw}$ is the hot wall area, $T_{hot}$ the hot wall temperature, $A_c$  the cross section area of the channel.
+
+The main equations to be solved for $\bar{U}$ and $T_o$ are [p_balance] and [q_balance]. These are non-linear, and can be easily solved with symbolic mathematics software or iterative solver
+packages such as `scipy`.
+
+!listing microreactors/gcmr/airjacket/re_estimate.py start= pqbalance( end=fsolve include-end=True
+
+Approximations for $T_m$ (which require an approximate heat transfer coefficient $h$) and $f$ using conventional correlations (such as Dittus-Boelter and Blasius correlations, respectively) can be used for an initial guess, but as the simulation is run, values are calculated directly from the results. With these results, the correct $f$ and $Nu$ can be determined for a given $U_m$.
+
+We have assumed uniform inlet ($T_{in}$) and hot-wall temperatures ($T_{wall}$) based on simple data from limiting conditions,
+while all remaining walls have adiabatic boundary conditions applied to temperature due to insulating material.
+
+The outlet was artificially extended by a few characteristic lengths and the Boussinesq acceleration term gradually ramped down
+to allow for the highly biased jet in the upper plenum to redistribute and flow outwards at the outlet uniformly in order to avoid any eddies
+that can cause numerical divergence-inducing backflow at the outlet. The gradual ramping down prevents any effect on the results upstream.
+Wall resolution was ensured by monitoring $y^+$1 values at the first Gauss Lobatto Legendre point off the wall.
+
+To capture the buoyant momentum source, we have applied the Boussinesq approximation. The Boussinesq term is a forcing term applied to the momentum Navier-Stokes
+Equation ([eq:ins]), which in non-dimensional terms, is expressed as
+\begin{equation}
+\tilde{F} = \frac{ \beta_0 (T - T_0) g L}{U ^2} \hat{z} = Ri \: \theta
+\end{equation}
+where $Ri$ is the Richardson number, $T$ the dimensional local temperature, and $\theta$ the non-dimensional local temperature.
```

Comment:
Theta does not feature in this equation?

Reply:
Please see the RHS on the extreme right: `Ri \: \theta`